### ISSUE TITLE
New Images are missing umount binary

### ISSUE THREAD

**Issue by TobiasGoerke** [datashim-io/datashim#192]
When deleting pods having mounted a volume provisioned by datashim, an error occurs in the csi-s3 container:

```bash
I0630 12:24:15.393382       1 utils.go:98] GRPC request: {"target_path":"/var/lib/kubelet/pods/xxx/volumes/kubernetes.io~csi/pvc-fxxx/mount","volume_id":"pvc-xxx"}
E0630 12:24:15.393504       1 utils.go:101] GRPC error: rpc error: code = Internal desc = unmount failed: exec: "umount": executable file not found in $PATH 
```

The issue seems to be the new distroless image, which is missing the umount binary. While this works (old image):

`docker run --rm -it --entrypoint umount quay.io/datashim/csi-s3:latest-amd64`

this does not (new image):

`docker run --rm -it --entrypoint umount quay.io/datashim-io/csi-s3:latest-amd64`

**Comment by srikumar003** [comment]
@TobiasGoerke thanks! I'll issue a fix soon

**Comment by srikumar003** [comment]
Uploaded new images and issued a fix with #192
